projects/angular/src/data/datagrid/utils/key-navigation-grid.controller.ts

```diff
@@ -176,7 +183,7 @@ export class KeyNavigationGridController implements OnDestroy {
     return this._activeCell;
   }
 
-  setActiveCell(activeCell: HTMLElement) {
+  setActiveCell(activeCell: HTMLElement, actionableItem = false) {
```

Comment:
```suggestion
  setActiveCell(activeCell: HTMLElement, { skipItemFocus } = { skipItemFocus: false }) {
```

The boolean parameter should be "named" by being passed as an object to promote readability of the code.